.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please remove this file.
The Vue CLI codebase was not formatted by prettier. Introducing prettier in an irrelevant PR could bring in unrelated diffs.

It is also the cause of the failing TypeScript test.

Reply:
PR updated